public/js/app/components/CourseSectionList.jsx

```diff
@@ -192,13 +200,15 @@ function CourseSectionList({ courseInfo = {}, partToShow, syllabus = {}, syllabu
         sectionId="Content"
         syllabusMarkerAriaLabel={translation.courseLabels.syllabus_marker_aria_label}
       />
+      <p>Here comes the course execution</p>
```

Comment:
this should be here right?

Reply:
Forgot to clean up after myself, thank you!